about.html

```diff
@@ -0,0 +1,179 @@
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/a36c13d2c3.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="./Style/about.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav class="top-nav">
+      <ul>
+        <li>
+          <a href="#"> <i class="fa fa-facebook"></i> </a>
+        </li>
+        <li>
+          <a href="#"> <i class="fa fa-twitter"></i> </a>
+        </li>
+        <li><a href="#"> English </a></li>
+        <li><a href="#"> My Page </a></li>
+        <li><a href="#" class="lastA"> Logout </a></li>
+      </ul>
+    </nav>
+
+    <nav class="web-nav">
+      <a href="index.html" class="brand"
+        ><img src="./images/logo.png" alt="brand-img"
+      /></a>
+      <ul>
+        <li><a href="about.html"> About </a></li>
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join </a></li>
+        <li><a href="#"> Sponsor </a></li>
+        <li><a href="#"> News </a></li>
+        <li><a href="#" class="lastCCA"> CC Campaign </a></li>
+      </ul>
+    </nav>
+
+    <nav class="mobile-nav">
+      <ul>
+        <li class="close_btn">
+          <i class="fa-solid fa-xmark" onclick="hamburger()"></i>
+        </li>
+        <li><a href="about.html"> About </a></li>
+        <li><a href="#"> Program </a></li>
+        <li><a href="#"> Join </a></li>
+        <li><a href="#"> Sponsor </a></li>
+        <li><a href="#"> News </a></li>
+        <li><a href="#"> CC Campaign </a></li>
+      </ul>
+    </nav>
+    <header class="Landing_Page">
+      <div id="hamburger">
+        <i class="fa-solid fa-bars" onclick="hamburger()"></i>
+      </div>
+      <p class="hello">"Hello! Sharing World"</p>
+      <h1>
+        CREATIVE COMMONS <br />
+        GLOBAL SUMMIT <br />
+        2015
+      </h1>
+      <p class="message">
+        The Creative Commons Global Summit , held every two years, is a place
+        where CC activists from all over world and experts, scholars and
+        activists from all walks of life who support and practice open culture
+        gather together. In particular, this year's CC Summit awaits the
+        participation of various organizations and individual participants who
+        want to participate in activities to spread the value of sharing, free
+        culture, and open knowledge with CC. Free software supporters, Wikipedia
+        activists, open knowledge foundation activists, art galleries,
+        libraries, museums, archives, governments, foundations, lawyers, social
+        activists, etc. various people active in the field of open content and
+        open knowledge gather in one place to share richer and more dynamic We
+        hope that it will be an opportunity to share our thoughts on creating an
+        ecosystem.
+      </p>
+
+      <p class="place">
```

Comment:
- [ ] Please fix the padding for the paragraph and make sure to stick with colors and font size for the title 
 
| Your design | Behance |
| ------------- | --------- |
| ![image](https://user-images.githubusercontent.com/74744787/163299863-75c24823-0005-4d2f-8d58-4d9c2b3b5eb3.png)![image](https://user-images.githubusercontent.com/74744787/163299952-c2baf712-8d69-404e-8d3b-94acc5024f1b.png) | ![image](https://user-images.githubusercontent.com/74744787/163300032-2af4feb9-04f0-48c9-ae11-07577b9ff696.png) |
| | notice here the paragraph don't have border |

Reply:
Thank you for mentioning these mistakes